doc/customizing_guide/units.rst

```diff
@@ -0,0 +1,71 @@
+Unit conversion in glue
+=======================
+
+.. note:: Support for automatic unit conversion in glue is experimental - at the moment
+          the ability to select units for the x and y axes is only available in the profile viewer.
+
+Data components can be assigned units as a string (or `None` to indicate no known units).
```

Comment:
This breaks backward compatibility -- See failures in https://github.com/spacetelescope/jdaviz/actions/runs/4053913069/jobs/6975103932

Reply:
Thanks, will have a think if there is a way we can avoid breaking backward compatibility